ts/features/lollipop/hooks/useLollipopLoginSource.tsx

```diff
@@ -67,11 +76,13 @@ export const useLollipopLoginSource = (loginUri?: string) => {
             }
           },
           publicKey: O.some(key)
-        });
-      })
-      .catch(_ => {
+        })
+      ),
+      TE.mapLeft(error => {
+        trackLollipopIdpLoginFailure(error.message);
```

Comment:
Here we are collection the error message coming from the generation of a brand new crypto key.  This case is not  explicitly written [here]( https://docs.google.com/spreadsheets/d/1bXNRcTDKU-5l2RqCcwSbT7Y1M9QQTWtS_f-3-AIqSYk/edit#gid=1624395933). It is the `reason` of the key generation events. Is that the wanted behavior? Or do you prefer to add a specific reason string also for this scenario?

Reply:
It is the desired behaviour. We did not report it into the Sheet since possible values are already tracked in the `LOLLIPOP_KEY_GENERATION_FAILURE` event but we can add a note (in the Sheet) to specify that it can also be "any of the `LOLLIPOP_KEY_GENERATION_FAILURE` event". Should we?

Reply:
Okay, it sounds good.